test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/scripts/multi-logger.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Would it possibly be better to generate these logs in go via a helper function in the test? It would be a lot easier to debug/update - and the code would be rather simple. 

I am not super familiar with the e2e framework so please let me know if there is a good reason I am not seeing.

Reply:
I used a script here since the original QA card used docker for the log generation service and I don't believe the framework has support for that.

I'll change it to be just go code instead 👍

Reply:
Awesome thanks! I think that would make things much simpler overall. 

definitely feel free to adapt the manual tests as you convert them to automated tests :)